AUTHORS

```diff
@@ -0,0 +1,20 @@
+# This is the list of Xilem's significant contributors.
```

Comment:
I considered it and had it in my initial draft. I decided against because:
* The Druid `AUTHORS` file isn't super representitive of *significant contributors*, as many of the actual top contributors are missing from it.
* We already state that for copyright look at source control, and we have that massive co-authored commit with a good list.
* The license doesn't mandate it, it's a voluntary step.

For my liking, having two lists with overlapping names seemed to cause more potential confusion than value.